mycroft/util/audio_test.py

```diff
@@ -80,6 +81,19 @@ def main():
         help="Add extra output regarding the recording")
     args = parser.parse_args()
 
+    config = Configuration.get()
+    if "device_name" in config["listener"]:
+        dev = config["listener"]["device_name"]
+    elif "device_index" in config["listenter"]:
+        dev = "Device at index {}".format(config["listener"]["device_index"])
+    else:
+        dev = "Default device"
+    samplerate = config["listener"]["sample_rate"]
+    play_cmd = config["play_wav_cmdline"].replace("%1", "WAV_FILE")
+    print(" ============================= Info ==============================")
```

Comment:
Nice!  Would also be nice to list all devices when someone is looking for a mic.  I'm going to add a -l option.

Reply:
Sounds good